dotnet-vnc/Dockerfile

```diff
@@ -6,4 +6,8 @@ RUN wget -q https://packages.microsoft.com/config/ubuntu/18.04/packages-microsof
     dpkg -i packages-microsoft-prod.deb && rm -rf packages-microsoft-prod.deb && \
     add-apt-repository universe && \
     apt-get update && apt-get -y -o APT::Install-Suggests="true" install dotnet-sdk-2.2 && \
```

Comment:
Oops, it seems like I forgot to update `dotnet-vnc/Dockerfile` too when I updated `dotnet/Dockerfile` to .NET Core 3.1 in 3f81f46b3a46721cd39739dcff427ec76d637bd4!

See also https://github.com/gitpod-io/workspace-images/pull/174